Lib/test/test_types.py

```diff
@@ -1195,7 +1185,8 @@ def test_union(self):
         other = {'c': 3, 'p': 0}
         self.assertDictEqual(view | other, {'a': 0, 'b': 1, 'c': 3, 'p': 0})
         self.assertDictEqual(other | view, {'c': 2, 'p': 0, 'a': 0, 'b': 1})
-        self.assertEqual(view, {'a': 0, 'b': 1, 'c': 2})
+        # Should be test after implementing MappingProxy's Comparable trait
+        # self.assertEqual(view, {'a': 0, 'b': 1, 'c': 2})
```

Comment:
```suggestion
        self.assertEqual(view, {'a': 0, 'b': 1, 'c': 2})
```
It is not desirable commenting out the codes inside the test and unmaking `expectedFailure`.